contracts/OpolisPay.sol

```diff
@@ -145,19 +144,25 @@ contract OpolisPay is ReentrancyGuard {
             )
         ) revert InvalidStake();
         if (memberId == 0) revert NotMember();
-        if (stakes[memberId]) revert DuplicateStake();
+
+        if (stakes[memberId] < 1){
+            stakes[memberId] = 1;
+        } else {
+            uint256 count = stakes[memberId];
+            stakes[memberId] += count;
+        }
```

Comment:
We only want to increment `stakes[memberId]` by 1 each time right? Otherwise each subsequent stakes doubles their count
```suggestion
        stakes[memberId] += 1;
```

Reply:
Cool, @c0rv0s the cleaner code to increment the stake count makes total sense. 

Will work on making the withdraw stakes work for multiple stakes per member. I think we probably just need to add another param to that function where someone can enter the stake number -- will require a front-end change and update to the subgraph. Let me know if you can think of an easier way.